packages/ui/src/walkthrough-card/WalkthroughCard.css

```diff
@@ -0,0 +1,45 @@
+.ui-WalkthroughCard--CardContainer {
+  display: flex;
+  border-radius: var(--layout-border-radius-xlarge);
+  box-shadow: var(--surface-shadow-medium), var(--surface-shadow-medium);
+  background-color: var(--surface-color-background-white);
+  max-width: var(--walkthrough-card-max-width);
+  flex-direction: column;
+  color: var(--support-border-color-info);
+  padding: var(--layout-padding-large);
+  border: var(--layout-border-width-none);
+  text-align: left;
+  transition: box-shadow var(--transition-medium);
+  text-decoration: none;
+}
+
+.ui-WalkthroughCard--Title {
+  color: var(--support-border-color-info);
+  font: var(--typography-bold-h4);
+  text-decoration: var(--walkthrough-card-title-text-decoration-hidden);
+  transition: text-decoration-color var(--transition-medium);
+  text-underline-offset: var(--walkthrough-card-title-text-underline-offset);
+}
+
+.ui-WalkthroughCard--CardContainer[data-hovered] {
+  cursor: pointer;
+  box-shadow: var(--surface-shadow-large), var(--surface-shadow-large);
+  border: 1px solid var(--surface-color-border-default);
```

Comment:
Good call on the border on hover. Completely missed that before. Can we add it as a transparent border by default and then just change the border color when hovering? I think the animation looks better that way. Also update the transition declaration to include boder-color.